docs/GettingStartedDocs/VisualStudioLinux.md

```diff
@@ -36,7 +36,7 @@ but a non-SGX machine can still be used:
 
 On the Linux build machine, or after opening an ssh session into the VM:
 
-- Install the Open Enclave SDK.  See [installation instructions for Ubuntu 18.04](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/install_oe_sdk-Ubuntu_18.04.md), [SGX1 instructions](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/Contributors/SGX1GettingStarted.md).
+- Install the Open Enclave SDK.  See [installation instructions for Ubuntu 18.04](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/install_oe_sdk-Ubuntu_18.04.md) or [installation instructions for Ubuntu 20.04](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/install_oe_sdk-Ubuntu_20.04.md) , [SGX1 instructions](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/Contributors/SGX1GettingStarted.md).
```

Comment:
This seems to originally have meant 1 or 2, so the change should be 1 or 2 or 3. Changing this to a list would make it easier to read and notice that.